randovania/games/dread/json_data/Artaria.txt

```diff
@@ -4838,13 +4843,14 @@ Extra - asset_id: collision_camera_081
               Walljump (Beginner) and Use Spin Boost
   > Under left side blocks
       Any of the following:
-          Lay Cross Bomb
-          All of the following:
-              Gravity Suit
-              Walljump (Beginner) or Simple IBJ or Use Spin Boost
+          Gravity Suit
+          Infinite Bomb Jump (Beginner) and Lay Cross Bomb
+          Water Bomb Jump (Intermediate) and Perform WBJ
           All of the following:
-              Water Bomb Jump (Intermediate) and Perform WBJ
-              Space Jump or Walljump (Beginner)
+              Before Artaria - Screw Attack Rotatable and Simple IBJ and Use Spin Boost
+              Any of the following:
+                  Morph Ball
+                  Screw Attack and Before Artaria - Screw Attack Rotatable
```

Comment:
This should be Simple IBJ or Spin Boost, not both. also second rotatable event is redundant.